data/hero.yml

```diff
@@ -1,8 +1,6 @@
 hero_text: "The largest gathering of Pythonistas in India for the Python
             programming language."
-primary_button:
-  text: Submit BoF or Poster Session
-  link: "https://in.pycon.org/cfp/bofs-and-posters-2021/proposals/"
+
 secondary_button:
```

Comment:
Should we make this button now primary button?

Reply:
+1